cob_default_robot_behavior/scripts/trigger_srvs_cob4-8.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python
```

Comment:
https://travis-ci.org/ipa320/cob_robots/jobs/240757407#L3205

install tag is missing for cob4-8 in https://github.com/ipa-nhg/cob_robots/blob/f25d19d215aae7ce4c99369478d5a37d10e68d21/cob_default_robot_behavior/CMakeLists.txt#L10-L14